Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -42,7 +42,7 @@ static async Task<List<DeviceCalendar>> PlatformGetCalendarsAsync()
                 });
             }
             cur.Dispose();
-            return calendars;
+            return calendars.AsEnumerable();
```

Comment:
Is this required - List is already IEnumerable

Reply:
Okay wasn't sure if I should make it explicit will revert those parts of the changes